detect_secrets/core/scan.py

```diff
@@ -206,7 +210,11 @@ def scan_for_allowlisted_secrets_in_file(filename: str) -> Generator[PotentialSe
     # know which lines we want to scan.
     try:
         for lines in _get_lines_from_file(filename):
-            yield from _scan_for_allowlisted_secrets_in_lines(enumerate(lines, start=1), filename)
+            lines_list = [
+                (number, value, False, False)
+                for number, value in list(enumerate(lines, start=1))
+            ]
+            yield from _scan_for_allowlisted_secrets_in_lines(lines_list, filename)
```

Comment:
I don't see that we're using is_added/is_removed in _scan_for_allowlisted_secrets_in_lines, is it necessary to pass those vars in ?